### PR TITLE
refactor: Migrate AssetHeader to React

### PR DESCRIPTION
**Pull request by ffflorian**

### CODE REVIEW

File: src/script/main/globals.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ import 'Components/groupList';
 import 'Components/guestModeToggle';
 import 'Components/icons';
 import 'Components/image';
-import 'Components/InfoToggle';
+import 'Components/infoToggle';
```
Comment (thisisamir98) [inline src/script/main/globals.ts]:
```suggestion
import 'Components/InfoToggle';
```

File: src/script/components/asset/AssetHeader.tsx [hunk 2]
```diff
@@ -17,28 +17,42 @@
...
+    </Fragment>
+  );
+};
+
+registerReactComponent('select-target', {
+  component: AssetHeader,
   template: `
     <span class="asset-header-name" data-bind="text: message_et.user().name(), css: message_et.accent_color"></span>
     <span class="asset-header-time" data-bind="text: timeText"></span>
   `,
```
Comment (thisisamir98) [inline src/script/components/asset/AssetHeader.tsx]: are we keeping this?

**Review by thisisamir98 — APPROVED** [review]

File: src/script/components/asset/AssetHeader.test.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    const message = new Message(createRandomUuid());
+    message.timestamp(timestamp);
+    message.user(user);
+
+    const assetHeader = new AssetHeaderPage({message});
+
+    expect(assetHeader.getUserName()).toBe(userName);
+
+    // TODO: Mock `formatTimeShort` to test this
+    // expect(assetHeader.getTime()).toBe('01/21 4:08 PM');
```
Comment (ffflorian) [inline src/script/components/asset/AssetHeader.test.tsx]: TODO

File: src/script/components/asset/AssetHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  message: Message;
+}
+
+const AssetHeader: React.FC<AssetHeaderProps> = ({message}) => {
+  const timestamp = message.timestamp();
+  const timeText = `${formatDayMonthNumeral(timestamp)} ${formatTimeShort(timestamp)}`;
+
+  return (
+    <Fragment>
+      <span className={cx('asset-header-name', message.accent_color)} data-uie-name="user-name">
```
Comment (bennycode) [inline src/script/components/asset/AssetHeader.tsx]:
```suggestion
      <span className={cx('asset-header-name', message.accent_color)} data-uie-name="asset-header-user-name">
```

File: src/script/components/asset/AssetHeader.tsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        {message.user().name()}
+      </span>
+      <span className="asset-header-time" data-uie-name="header-time">
```
Comment (bennycode) [inline src/script/components/asset/AssetHeader.tsx]:
```suggestion
      <span className="asset-header-time" data-uie-name="asset-header-time">
```

**Review by bennycode — CHANGES_REQUESTED** [review]
The `data-uie-name` value should be unique.

**Review by bennycode — APPROVED** [review]

**Pull request merged**